modules/virt-configuring-live-migration-limits.adoc

```diff
@@ -34,12 +34,14 @@ spec:
     parallelMigrationsPerCluster: 5 <3>
     parallelOutboundMigrationsPerNode: 2 <4>
     progressTimeout: 150 <5>
+    allowPostCopy: false <6>
 ----
 <1> Bandwidth limit of each migration, where the value is the quantity of bytes per second. For example, a value of `2048Mi` means 2048 MiB/s. Default: `0`, which is unlimited.
 <2> The migration is canceled if it has not completed in this time, in seconds per GiB of memory. For example, a VM with 6GiB memory times out if it has not completed migration in 4800 seconds. If the `Migration Method` is `BlockMigration`, the size of the migrating disks is included in the calculation.
 <3> Number of migrations running in parallel in the cluster. Default: `5`.
 <4> Maximum number of outbound migrations per node. Default: `2`.
 <5> The migration is canceled if memory copy fails to make progress in this time, in seconds. Default: `150`.
+<6> If a VM is running a heavy workload and the memory dirty rate is too high, this can prevent the migration from one node to another from converging. To prevent this, you can enable post copy mode. By default, `allowPostCopy` is set to `false`. 
```

Comment:
Nit (double "prevent"):
```suggestion
<6> If a VM is running a heavy workload and the memory dirty rate is too high, this can prevent the migration from one node to another from converging. To address this, you can enable post copy mode. By default, `allowPostCopy` is set to `false`. 
```